### PR TITLE
Fix RouteAuthGuard not to redirect to SignupPage when page is loaded

### PR DESCRIPTION
**Pull request by Harineko0**

### CODE REVIEW

File: one-pager-maker/src/RouteAuthGuard.tsx [hunk 1]
```diff
@@ -1,18 +1,36 @@
-import { Navigate, useLocation } from "react-router-dom";
+import {Navigate, useLocation} from "react-router-dom";
 import React from "react";
-import { useAppSelector } from "./redux/hooks";
+import {useAppSelector} from "./redux/hooks";
+import {Loader2} from "lucide-react";
```
Comment (binnmti) [inline one-pager-maker/src/RouteAuthGuard.tsx:4]: Why 2?

**Review by binnmti — APPROVED** [review]
LGTM

File: one-pager-maker/src/RouteAuthGuard.tsx (hunk 1 above)
Comment (Harineko0) [reply one-pager-maker/src/RouteAuthGuard.tsx:4]: Loader is dashed line circle

Comment (binnmti) [reply one-pager-maker/src/RouteAuthGuard.tsx:4]: I don't know. Why is it called `Loader2` for dashed line circle?

Comment (Harineko0) [reply one-pager-maker/src/RouteAuthGuard.tsx:4]: This is it!
https://lucide.dev/icons/loader
https://lucide.dev/icons/loader-circle
In lucide-react library, Loader2 is icons/loader-circle and Loader is icons/loader

**Pull request merged**